core/src/main/resources/hudson/security/GlobalSecurityConfiguration/index.groovy

```diff
@@ -26,6 +26,23 @@ l.layout(norefresh:true, permission:app.ADMINISTER, title:my.displayName, csscla
             set("descriptor", my.descriptor);
 
             f.optionalBlock( field:"useSecurity", title:_("Enable security"), checked:app.useSecurity) {
+                f.entry (title:_("Disable remember me"), field: "disableRememberMe") {
+                    f.checkbox()
+                }
+
+                f.entry(title:_("Access Control")) {
+                    table(style:"width:100%") {
+                        f.descriptorRadioList(title:_("Security Realm"),varName:"realm",         instance:app.securityRealm,         descriptors:SecurityRealm.all())
```

Comment:
@recena The lines [67-76](https://github.com/jenkinsci/jenkins/pull/2900/files#diff-459faeca0eeaad7c70ad86b5eb914055L67) were moved to [29-38](https://github.com/jenkinsci/jenkins/pull/2900/files#diff-459faeca0eeaad7c70ad86b5eb914055R29) without modification.

Hence the answer for your question…

> I don't understand this code formatting, why not [this other way]?

… is: "It has been that way before and left unchanged".